packages/communication-react/CHANGELOG.stable.md

```diff
@@ -10,24 +10,20 @@ Mon, 13 Jan 2025 19:28:45 GMT
 [Compare changes](https://github.com/azure/communication-ui-library/compare/1.22.0...1.23.0)
 
 ### Features
-- We are excited to announce that the Azure Communication Services Web UI Library now supports the Media Access feature. This feature allows organizers, co-organizers, and presenters to control the ability of other attendees to send audio and video while in a call. Attendees can determine if their audio or video is enabled or disabled and check the media access status of other participants. Developers can integrate this functionality through our composites (e.g., CallComposite, CallWithChatComposite) as well as through components. ([PR #5546](https://github.com/azure/communication-ui-library/pull/5546) by fuyan@microsoft.com)
-- Hard mute feature UI changes ([PR #5452](https://github.com/azure/communication-ui-library/pull/5452) by fuyan@microsoft.com)
+- We are excited to announce that the Azure Communication Services Web UI Library now supports the Media Access feature. This feature allows organizers, co-organizers, and presenters to control the ability of other attendees to send audio and video while in a call. Attendees can determine if their audio or video is enabled or disabled and check the media access status of other participants. Developers can integrate this functionality through our composites (e.g., CallComposite, CallWithChatComposite) as well as through components. ([PR #5546](https://github.com/azure/communication-ui-library/pull/5546) by fuyan@microsoft.com, [PR #5452](https://github.com/azure/communication-ui-library/pull/5452) by fuyan@microsoft.com, [PR #5383](https://github.com/azure/communication-ui-library/pull/5383) by fuyan@microsoft.com, [PR #5539](https://github.com/azure/communication-ui-library/pull/5539) by fuyan@microsoft.com, [PR #5347](https://github.com/azure/communication-ui-library/pull/5347) by fuyan@microsoft.com)
```

Comment:
Do we typically include all the PR links for the new feature? Please take a look at the past releases that announce a new feature and make sure we are doing the same thing.

Reply:
We do if you look at the previous stable changelog. I did remove PR related to tests/storybook